api/Element.json

```
@@ -3,6 +3,14 @@
     "Element": {
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element",
+        "spec_url": [
+          "https://w3c.github.io/pointerevents/#extensions-to-the-element-interface",
+          "https://w3c.github.io/pointerlock/#extensions-to-the-element-interface",
+          "https://fullscreen.spec.whatwg.org/#api",
+          "https://w3c.github.io/DOM-Parsing/#extensions-to-the-element-interface",
+          "https://drafts.csswg.org/cssom-view/#extension-to-the-element-interface",
+          "https://dom.spec.whatwg.org/#interface-element"
+        ],
```

Let's put the spec first that defines the interfaces, extensions following.

```suggestion
        "spec_url": [
          "https://dom.spec.whatwg.org/#interface-element",
          "https://w3c.github.io/pointerevents/#extensions-to-the-element-interface",
          "https://w3c.github.io/pointerlock/#extensions-to-the-element-interface",
          "https://fullscreen.spec.whatwg.org/#api",
          "https://w3c.github.io/DOM-Parsing/#extensions-to-the-element-interface",
          "https://drafts.csswg.org/cssom-view/#extension-to-the-element-interface"
        ],
```